.github/workflows/container-build.yml

```diff
@@ -0,0 +1,16 @@
+name: Build container image every change.
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - 'v*'
+
+
+jobs:
+  build:
+    name: Build
+    uses: jvanz/kubewarden-controller/.github/workflows/container-image.yml@main
```

Comment:
There are some references to my private repo in this PR. There are the action repo and the with the end-to-end tests. This should be changed _before_ merging this PR. I opened it as it is to see if the whole team agree on how we run the tests. Once we are ready, I'll create the repos inside the Kubewarden org, update the repo and merge the code.

Reply:
Once we create the repos, we will be able to version them. Therefore, I will update the reference to the version used. Removing the `main` ref from the files.